src/AltinnCore/Common/Services/Implementation/FormSILocalDev.cs

```diff
@@ -121,45 +120,6 @@ public object GetPrefill(string org, string service, Type type, int partyId, str
             }
         }
 
-        /// <summary>
-        /// This method serialized the form model and store it in test data folder based on serviceId and partyId
-        /// </summary>
-        /// <typeparam name="T">The input type</typeparam>
-        /// <param name="dataToSerialize">The data to serialize</param>
-        /// <param name="instanceId">The formId</param>
-        /// <param name="type">The type</param>
-        /// <param name="org">The Organization code for the service owner</param>
-        /// <param name="service">The service code for the current service</param>
-        /// <param name="partyId">The partyId</param>
-        /// <param name="dataId">The data id</param>
-        public async Task<Guid> SaveFormModel<T>(T dataToSerialize, Guid instanceId, Type type, string org, string service, int partyId, Guid dataId)
-        {
```

Comment:
@hannekot do you mean this method?. if so it's moved to the DataSI. We will slowly remove this FormSILocalDev once all the methods are moved to DataSI